pkg/security/probe/model.go

```diff
@@ -116,6 +116,11 @@ func (ev *Event) ResolveFilePath(f *model.FileEvent) string {
 		}
 		f.SetPathnameStr(path)
 	}
+
+	if strings.HasPrefix(f.PathnameStr, model.FilelessExecutionFilenamePrefix) {
```

Comment:
computeFilenameFromParts gets us to `memfd:` for the basename so that we can do `exec.file.name == "memfd:"`. This change here is so we can do `exec.file.path == "" `. We decided not to do `exec.file.path == "memfd:*" ` because the secl parsing expects exec.file.path to be an absolute path.